cypress/support/fragments/inventory/holdingsMove/inventoryInstancesMovement.js

```diff
@@ -1,4 +1,5 @@
-import { KeyValue, Button, DropdownMenu, Modal, HTML, Section } from '../../../../../interactors';
+import { KeyValue, Button, DropdownMenu, Modal, HTML, Section, including, Accordion } from '../../../../../interactors';
```

Comment:
'including' is defined but never used. Allowed unused vars must match /React/u

'Accordion' is defined but never used. Allowed unused vars must match /React/u

Reply:
fixed